internal/resources/google/compute_cost_component_helpers.go

```diff
@@ -30,6 +30,21 @@ type ContainerNodeConfig struct {
 	GuestAccelerators []*ComputeGuestAccelerator
 }
 
+/*
+For the values initialized in the struct below, please refer to the
+	sustained-use-discounts section of the GCP Documentation.
+
+size_SUD is set to 4 as the Usage level (% of month) is 4
+	to make sure we don't overshoot the array
+*/
+const size_SUD = 4
+
+type SudValues struct {
+	thresholds       [size_SUD]float64
+	rates            [size_SUD]float64
+	incrementalRates [size_SUD]float64
+}
```

Comment:
```suggestion
sudRateSize is set to 4 as the Usage level (% of month) is 4
	to make sure we don't overshoot the array
*/
```suggestion
const sudRateSize = 4

type sudRates struct {
	thresholds       [sudRateSize]float64
	rates            [sudRateSize]float64
	incrementalRates [sudRateSize]float64
}
```

Updating var names to be more consistent with the format we use elsewhere (camelCase)